### PR TITLE
add readme minio-distributed.yaml minio-standalone.yaml

### PR DESCRIPTION
**Pull request by zhaosimon**
Minio solution introduction.
minio standalone yaml
minio distributed yaml

### CODE REVIEW

File: deploy/solutions/minio/HwameiStor-MinIOSolutionReadMe.txt [hunk 1]
```diff
@@ -0,0 +1,185 @@
+
+HwameiStor-MinIO Cloud Native Solution
+
+1. Solution Introduction
```
Comment (windsonsea) [inline deploy/solutions/minio/HwameiStor-MinIOSolutionReadMe.txt:4]:
```suggestion
## Solution Introduction
```

File: deploy/solutions/minio/HwameiStor-MinIOSolutionReadMe.txt [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+
+HwameiStor is a high availability local storage system for cloud-native stateful workloads. It creates a local storage resource pool for centrally managing all disks such as HDD, SSD, and NVMe. It uses the CSI architecture to provide distributed services with local volumes and provides data persistence capabilities for stateful cloud-native workloads or components.
+
+MinIO is a High Performance Object Storage released under GNU Affero General Public License v3.0. It is API compatible with Amazon S3 cloud storage service. Use MinIO to build high performance infrastructure for machine learning, analytics and application data workloads.
```
Comment (windsonsea) [inline deploy/solutions/minio/HwameiStor-MinIOSolutionReadMe.txt:8]:
```suggestion
MinIO is a High Performance Object Storage released under GNU Affero General Public License v3.0. It provides APIs compatible with Amazon S3 cloud storage service. You can use MinIO to build high performance infrastructure for machine learning, analytics and application data workloads.
```

File: deploy/solutions/minio/HwameiStor-MinIOSolutionReadMe.txt [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+
+MinIO is used as the main storage for cloud-native applications. Compared with traditional object storage, cloud-native applications require higher throughput and lower latency. These are the performance indicators that MinIO can achieve, with read/write speeds of up to 183 GB/sec and 171 GB/sec.
```
Comment (windsonsea) [inline deploy/solutions/minio/HwameiStor-MinIOSolutionReadMe.txt:10]:
```suggestion
MinIO is used as the main storage for cloud-native applications. Compared with the traditional object storage, MinIO can provide higher throughput and lower latency required by cloud-native applications. For example, MinIO can achieve the read and write capacity of up to 183 GB and 171 GB per second.
```

File: deploy/solutions/minio/HwameiStor-MinIOSolutionReadMe.txt [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+
+The ultimate high performance of MinIO is inseparable from the underlying storage platform. Local storage has the highest read and write performance among many storage protocols, which can undoubtedly provide performance guarantee for MinIO. HwameiStor is a storage system that meets the requirements of the cloud-native era. It has the advantages of high performance, high availability, automation, low cost, and rapid deployment, and can replace expensive traditional SAN storage.
```
Comment (windsonsea) [inline deploy/solutions/minio/HwameiStor-MinIOSolutionReadMe.txt:12]:
```suggestion
The ultimate high performance of MinIO is inseparable from the underlying storage platform. Local storage is a good choice with the competitive read and write performance, which can undoubtedly provides performance guarantee for MinIO. HwameiStor is such a local storage system that waves with the cloud-native era. High performance, high availability, automation, low cost, rapid deployment, and more benefits attract you to replace those expensive and traditional storage schemes such as SAN (Storage Area Network).
```

File: deploy/solutions/minio/HwameiStor-MinIOSolutionReadMe.txt [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+
+
+2. Solution Validation
```
Comment (windsonsea) [inline deploy/solutions/minio/HwameiStor-MinIOSolutionReadMe.txt:15]:
```suggestion
## Solution Validation
```

File: deploy/solutions/minio/HwameiStor-MinIOSolutionReadMe.txt [hunk 6]
```diff
@@ -0,0 +1,185 @@
...
+
+View StorageClass status
+
+[root@k8s-10-6-163-52 minio-test1]# kubectl get sc
+NAME                         PROVISIONER         RECLAIMPOLICY   VOLUMEBINDINGMODE      ALLOWVOLUMEEXPANSION   AGE
+hwameistor-storage-lvm-hdd   lvm.hwameistor.io   Delete          WaitForFirstConsumer   true                   45h
+
+
+
+3. Deploy MinIO on Kubernetes
```
Comment (windsonsea) [inline deploy/solutions/minio/HwameiStor-MinIOSolutionReadMe.txt:99]:
```suggestion
## Deploy MinIO on Kubernetes
```

File: deploy/solutions/minio/HwameiStor-MinIOSolutionReadMe.txt [hunk 7]
```diff
@@ -0,0 +1,185 @@
...
+
+3.1 Standalone deployment
```
Comment (windsonsea) [inline deploy/solutions/minio/HwameiStor-MinIOSolutionReadMe.txt:101]:
```suggestion
### Standalone deployment
```

File: deploy/solutions/minio/HwameiStor-MinIOSolutionReadMe.txt [hunk 8]
```diff
@@ -0,0 +1,185 @@
...
+
+
+View PVCs on HwameiStor
+
+[root@k8s-10-6-163-52 minio-test1]# kubectl get pvc -nminio-2
+NAME      STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS                 AGE
+minio-2   Bound    pvc-3d4c1846-fc64-4af0-8104-64fc66c1c1bb   2Gi        RWO            hwameistor-storage-lvm-hdd   103m
+
+
+3.2 Distributed mode deployment
```
Comment (windsonsea) [inline deploy/solutions/minio/HwameiStor-MinIOSolutionReadMe.txt:144]:
```suggestion
### Distributed mode deployment
```

**Review by windsonsea — COMMENTED** [review]
We need not use 1.1, 2.1, 3.1 on a web page

File: deploy/solutions/minio/HwameiStor-MinIOSolutionReadMe.txt (hunk 4 above)
Comment (Michelle951) [reply deploy/solutions/minio/HwameiStor-MinIOSolutionReadMe.txt:12]: "Local storage has the highest read and write performance among many storage protocols, which can undoubtedly provide performance guarantee for MinIO."

Does it mean Hwameistor's Local Storage?

File: deploy/solutions/minio/HwameiStor-MinIOSolutionReadMe.txt [hunk 9]
```diff
@@ -0,0 +1,185 @@
...
+NAME                         PROVISIONER         RECLAIMPOLICY   VOLUMEBINDINGMODE      ALLOWVOLUMEEXPANSION   AGE
+hwameistor-storage-lvm-hdd   lvm.hwameistor.io   Delete          WaitForFirstConsumer   true                   45h
+
+
+
+3. Deploy MinIO on Kubernetes
+
+3.1 Standalone deployment
+
+Deploy based on official helm chart and install MinIO chart.
```
Comment (Michelle951) [inline deploy/solutions/minio/HwameiStor-MinIOSolutionReadMe.txt:103]:
```suggestion
Deploy based on the official helm chart and install the MinIO chart.
```

File: deploy/solutions/minio/HwameiStor-MinIOSolutionReadMe.txt [hunk 10]
```diff
@@ -0,0 +1,185 @@
...
+
+[root@k8s-10-6-163-52 minio-test1]# helm repo add minio https://helm.min.io
+
+[root@k8s-10-6-163-52 minio-test1]# helm repo list | grep minio
+minio         	https://helm.min.io/
+
+
+Standalone mode deployment.
```
Comment (Michelle951) [inline deploy/solutions/minio/HwameiStor-MinIOSolutionReadMe.txt:111]:
```suggestion
Standalone mode deployment
```
for consistency

**Comment by sun7927** [comment]
tracked by another PR, closed

**Pull request closed without merging**